charts/observability-pipelines-worker/values.yaml

```diff
@@ -0,0 +1,340 @@
+  pullPolicy: IfNotPresent
+  # image.pullSecrets -- Specify the
+  # [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+  pullSecrets: []
+  #  - name: <REGISTRY_SECRET>
+
+# replicas -- Specify the number of replicas to create.
+replicas: 1
+
+# podManagementPolicy -- Specify the
+# [podManagementPolicy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies).
+podManagementPolicy: OrderedReady
+
+## TODO: Support Watermark Pod Autoscaler?
+autoscaling:
+  # autoscaling.enabled -- If **true**, create a
+  # [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+  enabled: false
+  # autoscaling.minReplicas -- Specify the minimum number of replicas.
+  minReplicas: 1
+  # autoscaling.maxReplicas -- Specify the maximum number of replicas.
+  maxReplicas: 10
+  # autoscaling.targetCPUUtilizationPercentage -- Specify the target CPU utilization.
+  targetCPUUtilizationPercentage: 80
+  # autoscaling.targetMemoryUtilizationPercentage -- (int) Specify the target memory utilization.
+  targetMemoryUtilizationPercentage:
+  # autoscaling.behavior -- Configure separate scale-up and scale-down behaviors.
+  behavior: {}
+    # scaleDown:
+    #   stabilizationWindowSeconds: 300
+
+podDisruptionBudget:
+  # podDisruptionBudget.enabled -- If **true**, create a
+  # [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+  enabled: false
+  # podDisruptionBudget.minAvailable -- Specify the number of Pods that must still be available after an eviction.
+  minAvailable: 1
+  # podDisruptionBudget.maxUnavailable -- (int) Specify the number of Pods that can be unavailable after an eviction.
+  maxUnavailable:
+
+serviceAccount:
+  # serviceAccount.create -- If **true**, create a ServiceAccount.
+  create: true
+  # serviceAccount.name -- Specify a preexisting ServiceAccount to use if `serviceAccount.create` is **false**.
+  name: default
+  # serviceAccount.annotations -- Annotations to add to the ServiceAccount, if `serviceAccount.create` is **true**.
+  annotations: {}
+
+# podAnnotations -- Set annotations on Pods.
+podAnnotations: {}
+
+# podLabels -- Set labels on Pods.
+podLabels: {}
+
+# podPriorityClassName -- Set the
+# [priorityClassName](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass).
+podPriorityClassName: ""
+
+# podHostNetwork -- Enable the hostNetwork option on Pods.
+podHostNetwork: false
+
+# podSecurityContext -- Allows you to overwrite the default
+# [PodSecurityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).
+podSecurityContext: {}
+
+# securityContext -- Specify securityContext for containers.
+securityContext: {}
+
+# command -- Override default image command.
+command: []
+
+# args -- Override default image arguments.
+args:
+  - --config
```

Comment:
Always wondered this- would it be reasonable to use `--config-dir` instead so it's easier for folks to mount additional config files?

Reply:
😆 take it up with @lukesteensen! Currently OPW just takes a single config file, https://github.com/DataDog/observability-pipelines-worker/blob/master/src/main.rs#L11

Reply:
Following up on this @silversupreme - this was a "v0.1.0" intentional limitation we'll come back to down the line.

Reply:
Sorry, thought I'd already responded to this @silversupreme! But yeah, we're intentionally scoping down the wide range of config loading options from Vector, at least for now. I think `--config-dir` specifically is one of the more complex ones, but we're happy to add support if we really think it'll be valuable to users.